src/legacy/core_plugins/interpreter/index.ts

```diff
@@ -28,7 +28,16 @@ export default function InterpreterPlugin(kibana: any) {
     require: ['kibana', 'elasticsearch'],
     publicDir: resolve(__dirname, 'public'),
     uiExports: {
-      injectDefaultVars: server => ({ serverBasePath: server.config().get('server.basePath') }),
+      injectDefaultVars: server => ({
+        serverBasePath: server.config().get('server.basePath'),
+        interpreterConfig: server.config().get('interpreter'),
+      }),
+    },
+    config: (Joi: any) => {
+      return Joi.object({
+        enabled: Joi.boolean().default(true),
+        enableInVisualize: Joi.boolean().default(true),
```

Comment:
Are we good to make `true` the default now? I assume that's why there's the "do not merge" warning 😃

Reply:
yup. if i understood @rayafratkina correctly we are good to merge this however, next week we'll have a discussion if we need to switch the default to true and there is plenty of time to switch it then.